docs/book/src/user/concepts.md

```diff
@@ -5,73 +5,74 @@
+A source of computational resources, such as Machines and networking. For example, cloud Infrastructure Providers incluede AWS, Azure, and Google, and bare metal Infrastructure Providers include VMware, MAAS, and metal3.io.
+
+When there is more than one way to obtain resources from the same Infrastructure Provider (such as AWS offering both EC2 and EKS), each way is referred to as a variant.
 
 ### Bootstrap provider
 
-The bootstrap provider is responsible for (usually by generating cloud-init or similar):
+The Bootstrap Provider is responsible for:
 
 1. Generating the cluster certificates, if not otherwise specified
 1. Initializing the control plane, and gating the creation of other nodes until it is complete
 1. Joining master and worker nodes to the cluster
 
 ### Control plane
 
-The control plane (sometimes referred to as master nodes) is a set of [services](https://kubernetes.io/docs/concepts/#kubernetes-control-plane) that serve the Kubernetes API and reconcile desired state through the control-loops.
+The control plane is a set of [services](https://kubernetes.io/docs/concepts/#kubernetes-control-plane) that serve the Kubernetes API and continuously reconcile desired state using control loops.
+
+* __Machine-based__ control planes are the most common type, used by tools like kubeadm and kubespray. Dedicated machines are provisioned, running [static pods](https://kubernetes.io/docs/tasks/configure-pod-container/static-pod/) for components such as [kube-apiserver](https://kubernetes.io/docs/admin/kube-apiserver/), [kube-controller-manager](https://kubernetes.io/docs/admin/kube-controller-manager/) and [kube-scheduler](https://kubernetes.io/docs/admin/kube-scheduler/).
 
-* __Machine Based__ based control planes are the most common type deployment model and is used by tools like kubeadm and kubespray. Dedicated machines are provisioned running [*static pods*](https://kubernetes.io/docs/tasks/configure-pod-container/static-pod/) for the control plane components such as  [*kube-apiserver*](https://kubernetes.io/docs/admin/kube-apiserver/), [*kube-controller-manager*](https://kubernetes.io/docs/admin/kube-controller-manager/) and [*kube-scheduler*](https://kubernetes.io/docs/admin/kube-scheduler/).
+* __Pod-based__ deployments require an external hosting cluster. The control plane components are deployed using standard *Deployment* and *StatefulSet* objects and the API is exposed using a *Service*.
 
-* __Pod Based__  deployments require an external hosting cluster, the control plane is deployed using standard *Deployment* and *StatefulSet* objects and then the API exposed using a *Service*.
+* __External__ control planes are offered and controlled by some system other than Cluster API, such as GKE, AKS, EKS, or IKS.
 
-* __External__ control planes are offered and controlled by some system other than Cluster API (e.g., GKE, AKS, EKS, IKS).
+As of v1alpha2, __machine-based__ is the only control plane type that Cluster API supports.
```

Comment:
Should this capitalisation match the title above? 
```suggestion
As of v1alpha2, __Machine-Based__ is the only control plane type that Cluster API supports.
```

Reply:
I'm wondering if we should update this language to indicate that other control plane types are now possible through the use of "Control Plane providers"?